packages/kununu-footer/index.jsx

```diff
@@ -10,11 +10,22 @@ import styles from './index.scss';
 import TuvIcon from './Tuv';
 
 export default class Footer extends Component { // eslint-disable-line
+  activeCountry = () => {
+    const {items: {countrySwitcher}} = this.props;
+    const active = countrySwitcher.filter(item => item.active);
```

Comment:
how can i improve it? active is being returned inside an array, so I need to access `active[0].value`

Reply:
Perhaps setting active[0] would make it a little nicer? Otherwise I'm not sure.. I think it's ok. But what if the active is not set? Before we were setting the first item.